File: java/twostones.java
import java.util.Scanner;

public class twostones {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int N = scan.nextInt();
		System.out.println(N % 2 == 0 ? "Bob" : "Alice");
		scan.close();
	}
}

File: ts/src/twostones.ts
import readline = require('readline')

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    const stones = parseInt(line)
    console.log(stones % 2 === 0 ? 'Bob' : 'Alice')
  })
  .on('close', () => {
    process.exit(0)
  })
